src/components/nodes-table/index.vue

```diff
@@ -33,16 +37,29 @@
             </template>
             <v-card>
               <v-card-text>
-                <v-checkbox
-                  v-for="col in managedNodes.allTableHeaders"
-                  :key="col.value"
-                  :value="col.value"
-                  hide-details
-                  :label="col.text"
-                  :input-value="managedNodes.columns"
-                  @change="managedNodes.columns = $event"
-                ></v-checkbox>
+                <draggable v-model="managedNodes.tableColumns">
+                  <v-checkbox
+                    v-for="col in managedNodes.tableColumns"
+                    :key="col.name"
+                    v-model="col.visible"
+                    :value="col.visible"
+                    hide-details
+                    :label="managedNodes.propDefs[col.name].label"
+                    :input-value="col.visible"
+                    false-value="false"
```

Comment:
doesn't this bind the false to `'false'` instead of `false`?

Reply:
I'll remove this, was a left-over from getting null or undefined when unchecked.